docs/orgs.md

```diff
@@ -43,6 +44,15 @@ This will create the organization data in a file `group-<snyk_group_id>-gitlab-o
 
 This will create the organization data in a file `group-<snyk_group_id>-bitbucket-server-orgs.json`
 
+
+## Bitbucket Cloud
```

Comment:
can you provide the url of the cloud bbc? SO the customer can self determine if they are using Cloud vs Server

Reply:
Done